exonum-java-binding-core/rust/integration_tests/src/executor.rs

```diff
@@ -50,6 +54,18 @@ pub fn check_detached(vm: &JavaVM) {
     assert!(!is_attached(vm));
 }
 
+/// Creates MainExecutor and initializes JNI cache
+pub fn create_executor_with_cache_initialized(vm: Arc<JavaVM>) -> MainExecutor {
```

Comment:
I was trying to come up with a better name, but came to a realisation that I don't understand why this is done when the executor is created and not a VM (and, therefore, requires a lengthy name)? The jni cache, as a vm, is supposed to be initialized whenever EJB classes are needed (with fakes), and only once, but you can create many executors and they don't imply a need for a cache.

Wouldn't it be better to initialize the cache once a VM is created _with_ fakes (= with efb-fakes classpath) in 
`fn create_vm(debug: bool, with_fakes: bool) -> JavaVM`?

Reply:
Looks reasonable

Reply:
Implemented. The one drawback is that it turned out `JavaVM#new()` detaches current thread just after VM creation and we have to re-attach it, but I believe in the context of integration tests this should not be a problem.